packages/plugin-core/src/commands/NoteLookupCommand.ts

```diff
@@ -172,7 +173,6 @@ export class NoteLookupCommand
 
       await this.provider.onUpdatePickerItems({
         picker: this._quickPick,
-        token: this.controller.createCancelSource().token,
```

Comment:
What was triggering the cancel?

Reply:
on pressing `tab`, this new Cancel source was created resulting in hijacking the token of Controller